lithops/standalone/standalone.py

```diff
@@ -40,18 +40,20 @@ class StandaloneHandler:
     """
 
     def __init__(self, standalone_config):
+        self.log_active = logger.getEffectiveLevel() != logging.WARNING
         self.config = standalone_config
         self.backend_name = self.config['backend']
         self.runtime = self.config['runtime']
 
+        self.start_timeout = self.config.get('start_timeout', 300)
+
+        self.auto_dismantle = self.config.get('auto_dismantle')
+        self.hard_dismantle_timeout = self.config.get('hard_dismantle_timeout')
```

Comment:
why the defaults been removed from here?

Reply:
defaults are in config.py. at this point there are always values set